.github/workflows/release.yml

```diff
@@ -1,6 +1,17 @@
 on:
-  repository_dispatch:
-    types: [release]
+  workflow_dispatch:
```

Comment:
How do we actually trigger this workflow ?

Reply:
It's gonna show up here: https://github.com/getsentry/snuba/actions?query=workflow%3A.github%2Fworkflows%2Frelease.yml